psst-gui/Cargo.toml

```diff
@@ -37,6 +37,7 @@ url = "2.2"
 
 [target.'cfg(windows)'.build-dependencies]
 winres = "0.1.11"
+ico = "0.1"
```

Comment:
FWIW, considering also that it's already in the dependency tree and depends on up-to-date dependencies, image 0.24 supports generating an .ico containing multiple variants (sizes):
https://docs.rs/image/latest/image/codecs/ico/struct.IcoEncoder.html#method.encode_images

Reply:
Updated to image-rs